### ISSUE TITLE
Ark Nova - changes in dark mode

### ISSUE THREAD

**Issue by rstanculescu** [FlavienBusseuil/bga-chrome-extension#74]
Today I saw three changes in dark mode in Ark Nova:

1. the map itself has a dark overlay, aside from the enclosures. it's not annoying, I'm just not sure if it was intended. feels like the rest of the displays (card display, association board) also have a dark overlay on them too now that I look closer.
2. when playing Animals or Sponsors nothing is grayed out anymore (cards in the display if my action isn't upgraded, or unplayable cards in hand due to money/strength/being the wrong type). probably same issue with Association, but I don't have a worker to use in my current game.
3. upgraded actions in the action bar no longer show with a different color. I have build upgraded but they all show regular black and white on a gray surrounding background. the unusable actions do have a darker overlay to show they can't be used, but not sure if they'd show color if upgrade, since I don't have such an action to test.

**Comment by cdelaforge** [comment]
Thanks so much for your feedback! I don't have the time to test all the games one by one, so these comments are invaluable to me.
Concerning the overlay, you're right: I added that recently because I thought the colors were too aggressive for a dark mode. If you see something that doesn't work, or that can be improved, don't hesitate to tell me, and if possible add a small screenshot so I can see what it is.
As for the rest, I think I've corrected it (see attached screenshot), but it'll be in the next version.
Don't hesitate to let me know about any other problems you see with this or other games!

![image](https://github.com/user-attachments/assets/828a36b1-8850-4564-bfd1-c95aace08fed)
